### PR TITLE
unausgeschlafen quefrency keymap

### PR DESCRIPTION
**Pull request by unausgeschlafen**
This is my keymap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/quefrency/keymaps/unausgeschlafen/keymap.c [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BASE 0
+#define _FN1 1
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+};
+
+#define _______ KC_TRNS
```
Comment (drashna) [inline keyboards/keebio/quefrency/keymaps/unausgeschlafen/keymap.c]: These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

File: keyboards/keebio/quefrency/keymaps/unausgeschlafen/keymap.c [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+#define XXXXXXX KC_NO
```
Comment (drashna) [inline keyboards/keebio/quefrency/keymaps/unausgeschlafen/keymap.c]: These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

File: keyboards/keebio/quefrency/keymaps/unausgeschlafen/rules.mk [hunk 3]
```diff
@@ -0,0 +1,2 @@
+BOOTLOADER = atmel-dfu # comment this line if you use arduinos
+DISABLE_JTAG = yes
```
Comment (drashna) [inline keyboards/keebio/quefrency/keymaps/unausgeschlafen/rules.mk]: This isn't actually needed. 

If you're on a recent version of QMK, JTAG is always disabled, now. 

```suggestion
```

**Comment by unausgeschlafen** [comment]
Hi @drashna,
thank you for the review. I implemented the proposed changes, reflashed my board and are typing this message. Everything seems to work perfectly.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**